docsource/090_contribute.rst

```diff
@@ -10,6 +10,85 @@ In order to contribute to the OpenUpgrade project, please
   database, and give back any code improvements developed during the project.
 
 
+How to contribute new migration scripts
+---------------------------------------
+
+To Contribute to OpenUpgrade you must make one pull request per module.
+
+For example, if you want to propose the migration script of the ``account``
+module from version 15.0 to version 16.0:
+
+  * Always ensure that there is no work already in progress by a community member.
+    For that purpose, go the issue named "Migration to version NN.0" on GitHub, where NN is the version for which you want to propose the script.
+    `here <https://github.com/OCA/OpenUpgrade/issues/3681/>`_.
```

Comment:
Are we going to keep this issue reference? It won't go well through time, when we were on Odoo 20 (or just already in 17).

Reply:
Well, the sentence before is "For example, if you want to propose the migration script of the ``account``
module from version 15.0 to version 16.0" so it make sense.
but well, replaced by "https://github.com/OCA/OpenUpgrade/issues?q=is%3Aopen++is%3Aissue+%22Migration+to+version%22+"